frontend/src/components/ShoesList.vue

```diff
@@ -0,0 +1,148 @@
+        <div class="col-2 d-flex flex-column align-items-end">
+          <label>Marca <input type="text" v-model="brand"></label>
+          <label>Nome <input type="text" v-model="name"></label>
+          <label>Cor <input type="text" v-model="color"></label>
+        </div>
+        <div class="col-2 d-flex flex-column align-items-end">
+          <label>Ref <input type="text" v-model="ref"></label>
+          <label>Material <input type="text" v-model="material"></label>
+          <label>Tamanho <input type="number" v-model="size"></label>
+        </div>
+        <div class="col-2 d-flex flex-column align-items-end">
+          <label>Peso <input type="number" v-model="weight"></label>
+          <label>Quantidade <input type="number" v-model="quantity"></label>
+          <label>Lucro %<input type="number" v-model="tax"></label>
+        </div>
+        <div class="col-2 col-offset-2 d-flex flex-column align-items-end">
+          <label>Preço de Custo <input type="number" v-model="net_price"></label>
+          <label>Detalhes <input type="text" v-model="description"></label>
+        </div>
+      </div>
+      <div>
+        <input
+          class="btn btn-primary"
+          type="submit"
+          value="Criar"
+          @click="addShoe(
+          { brand: brand, name: name, color: color, ref: ref, material: material, size: size,
+                 weight: weight, quantity: quantity, tax: tax, net_price: net_price, description: description
+             })"
+          :disabled="!name || !brand || !ref || !material || !color || !size || !quantity || !net_price || !tax">
+        <hr/>
+      </div>
+      <b-modal id="modalCreate" size="md"  title="Create" v-model="shoes.show">
+        <template #shoe> Criar </template>
+        <div class="d-block text-center">
+          <h3>Calçado cadastrado com sucesso!</h3>
+        </div>
+        <div slot="modal-footer" class="">
+          <b-button class="mt-3" variant="primary" @click="shoes.show=false" value="Fechar">Fechar</b-button>
+        </div>
+      </b-modal>
+    </div>
+
+    <div>
+      <p>Filtros</p><br/>
+      <label class="mr-2">Marca <input type="text" v-model="shoes.search_brand"></label>
+      <label class="mr-2">Nome <input type="text" v-model="shoes.search_name"></label>
+      <input
+          class="btn btn-primary"
+          type="submit"
+          value="Filtrar"
+          @click="getShoes(shoes)">
+      <hr/>
+    </div>
+
+    <div>
+      <p class="alert-danger" v-if="shoes_list.length === 0">Nenhum calçado cadastrado</p>
+      <table class="table" v-if="shoes_list.length != 0">
+        <thead class="thead-dark">
+          <tr>
+            <th scope="col">Item</th>
+            <th scope="col">Marca</th>
+            <th scope="col">Nome</th>
+            <th scope="col">Cor</th>
+            <th scope="col">Tamanho</th>
+            <th scope="col">Quantidade</th>
+            <th scope="col">Taxa %</th>
```

Comment:
Typo ou existe alguma lógica aqui que não entendemos ?
```suggestion
            <th scope="col">Lucro %</th>
```

Reply:
Foi Typo.